### PR TITLE
draft repo setup

### PR DESCRIPTION
**Pull request by sstevens2**

### CODE REVIEW

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -135,6 +137,80 @@ For that, we will use a tool called Markdown.
 > {: .solution }
 {: .challenge }
 
+
+## Setting up a repository
```
Comment (sstevens2) [inline _episodes/01-introduction.md]:
```suggestion
## Setting Up a Repository
```

File: _episodes/01-introduction.md [hunk 2]
```diff
@@ -135,6 +137,80 @@ For that, we will use a tool called Markdown.
...
+![Blank new repository page](../fig/blank_new_repo.png)
+
+In this lesson, we will be working on a general group website.
+You can imagine this website may be for your lab group, a specific project group,
+or another group you work with.
+Under the "Repository name" field type `group-website`.
+
+![Repository name set to group-website](../fig/set_repo_name.png)
+
+So we know what this repo is when we find it later we can also add a description.
```
Comment (unode) [inline _episodes/01-introduction.md]:
```suggestion
So we know what this repo is when we find it later, we can also add a description.
```

**Review by unode — APPROVED** [review]

**Review by JulianKarlBauer — APPROVED** [review]
Great, I like the detailed steps.

One thing that, from my perspective, could be improved, is the size of the pictures.
Can we shrink them a bit?
Currently the font size in the pictures is a larger than your instructions.



Maybe an alternative improvement to keep the instructions on a similar perception-level would be a frame/box around the pictures?

**Comment by tobyhodges** [comment]
@JulianKarlBauer I agree but suggest this is an issue that can be cleared up later in a separate PR. Especially as we're likely to have more screenshots in the other episodes by the time we've finished writing... 

Could you open an issue so we remember to come back to this, please?

File: _episodes/01-introduction.md (hunk 2 above)
Comment (JulianKarlBauer) [reply _episodes/01-introduction.md]: For me as non-native speaker, this sentence is hard to understand.
Does the following have the same meaning?
"To tell our future-self what this repo is about, we can also add a description."
"To know what this repo is about when we find it later, we can also add a description. "

Comment (sstevens2) [reply _episodes/01-introduction.md]: Maybe we should flip the order of the phrases.  Perhaps...
```suggestion
We can also add a description so we know what this repo is when we find it again after the workshop.
```
would be better?

Comment (tobyhodges) [reply _episodes/01-introduction.md]: I like this but wonder if it would be even clearer if it read "[...] when we find it **again** after the workshop."

**Review by tobyhodges — APPROVED** [review]

**Review by annefou — APPROVED** [review]
This is really great! I think it can be merged now. Thanks.

**Pull request merged**